demos/common/export_models/export_model.py

```diff
@@ -0,0 +1,361 @@
+            convert_tokenizer_command = "convert_tokenizer --utf8_replace_mode replace --with-detokenizer --skip-special-tokens --streaming-detokenizer -o {} {}".format(llm_model_path, source_model) 
+            if (os.system(convert_tokenizer_command)):
+                raise ValueError("Failed to export tokenizer model", source_model)
+    os.makedirs(os.path.join(model_repository_path, model_name), exist_ok=True)
+    gtemplate = jinja2.Environment(loader=jinja2.BaseLoader).from_string(text_generation_graph_template)
+    graph_content = gtemplate.render(tokenizer_model="{}_tokenizer_model".format(model_name), embeddings_model="{}_embeddings_model".format(model_name), model_path=model_path, **task_parameters)
+    with open(os.path.join(model_repository_path, model_name, 'graph.pbtxt'), 'w') as f:
+        f.write(graph_content)
+    print("Created graph {}".format(os.path.join(model_repository_path, model_name, 'graph.pbtxt')))
+    add_servable2config(config_file_path, model_name, os.path.relpath( os.path.join(model_repository_path, model_name), os.path.dirname(config_file_path)))
+    
+    
+def export_embeddings_model(model_repository_path, source_model, model_name, precision, task_parameters, version, config_file_path):
+    if os.path.isfile(os.path.join(source_model, 'openvino_model.xml')):
+            print("OV model is source folder. Skipping conversion.")
+            os.makedirs(os.path.join(model_repository_path, model_name, 'embeddings', version), exist_ok=True)
+            os.makedirs(os.path.join(model_repository_path, model_name, 'tokenizer', version), exist_ok=True)
+            shutil.move(os.path.join(source_model, 'openvino_tokenizer.xml'), os.path.join(model_repository_path, model_name, 'tokenizer', version, 'model.xml'))
+            shutil.move(os.path.join(source_model, 'openvino_tokenizer.bin'), os.path.join(model_repository_path, model_name, 'tokenizer', version, 'model.bin'))
+            shutil.move(os.path.join(source_model, 'openvino_model.xml'), os.path.join(model_repository_path, model_name, 'embeddings', version, 'model.xml'))
+            shutil.move(os.path.join(source_model, 'openvino_model.bin'), os.path.join(model_repository_path, model_name, 'embeddings', version, 'model.bin'))
+    else: # assume HF model name
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            embeddings_path = os.path.join(model_repository_path, model_name,'embeddings', version)
+            print("Exporting embeddings model to ",embeddings_path)
+            if not os.path.isdir(embeddings_path) or args['overwrite_models']:
+                optimum_command = "optimum-cli export openvino --disable-convert-tokenizer --model {} --task feature-extraction --weight-format {} --trust-remote-code --library sentence_transformers {}".format(source_model, precision, tmpdirname)
+                if (os.system(optimum_command)):
+                    raise ValueError("Failed to export embeddings model", source_model)
```

Comment:
Can we somehow get the stderr of the optimum command? It would give us better debugging capabilities. 
For example we will quickly recognize that not all embeddings mode support `--library sentence_transformers` parameter (I found this during development, but cant remember which combination it was).
In this case we will have optimum-intel error in stderr not being forwarded to this script @dtrawins

Reply:
errors from optimum are displayed on the console via optimum cli. Error messages should be visible to the user